mm/2s2h/Enhancements/Dialogue/AutoBombersCode.cpp

```diff
@@ -0,0 +1,25 @@
+#include "public/bridge/consolevariablebridge.h"
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/ShipInit.hpp"
+#include "global.h"
+
+#define CVAR_NAME "gEnhancements.Dialogue.AutoBombersCode"
+#define CVAR CVarGetInteger(CVAR_NAME, 0)
+
+extern "C" {
+void Font_LoadCharNES(PlayState* play, u8 codePointIndex, s32 offset);
+}
+
+void RegisterAutoBombersCode() {
+    COND_ID_HOOK(OnActorUpdate, ACTOR_EN_BOMBERS2, CVAR, [](Actor* actor) {
+        if (gPlayState->msgCtx.currentTextId == 0x729 && CHECK_QUEST_ITEM(QUEST_BOMBERS_NOTEBOOK)) {
```

Comment:
Text IDs are 4 bytes wide, and the game source puts leading zeros to have them all written as 4 bytes. Lets do the same for consistency.
```suggestion
        if (gPlayState->msgCtx.currentTextId == 0x0729 && CHECK_QUEST_ITEM(QUEST_BOMBERS_NOTEBOOK)) {
```